From a8c7c38c841cf72dfb828623e020968e30ed16fd Mon Sep 17 00:00:00 2001
From: Adam Ford <aford173@gmail.com>
Date: Tue, 23 Jun 2020 09:40:46 -0500
Subject: [PATCH 41/42] arm64: dts: beacon-imx8mm: Enable camera reset pins

There are two GPIO pins assigned to the camera and pin muxed.
Unfortunately, only one of these two pins was being used.  The
reset pin wasn't being claimed which caused the dmesg to throw
some extra warnings.

This patch assigns the reset pin to the correct GPIO and eliminates
the message.

Signed-off-by: Adam Ford <aford173@gmail.com>
---
 arch/arm64/boot/dts/freescale/beacon-imx8mm-baseboard.dtsi | 1 +
 1 file changed, 1 insertion(+)

diff --git a/arch/arm64/boot/dts/freescale/beacon-imx8mm-baseboard.dtsi b/arch/arm64/boot/dts/freescale/beacon-imx8mm-baseboard.dtsi
index d450a60cad51..8f32aafeddb9 100644
--- a/arch/arm64/boot/dts/freescale/beacon-imx8mm-baseboard.dtsi
+++ b/arch/arm64/boot/dts/freescale/beacon-imx8mm-baseboard.dtsi
@@ -374,6 +374,7 @@
 		AVDD-supply = <&reg_camera>;
 		DVDD-supply = <&reg_camera>;
 		pwn-gpios = <&gpio1 7 GPIO_ACTIVE_HIGH>;
+		rst-gpios = <&gpio1 6 GPIO_ACTIVE_LOW>;
 		mclk = <24000000>;
 		mclk_source = <0>;
 		port {
-- 
2.17.1

